src/debug/clean-pending-payments-for-dead-node.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * how to run:
+ *	. ./.envrc && PUBKEY="" \
+ *    yarn ts-node \
+ *		--files \
+ *			-r tsconfig-paths/register \
+ *			-r src/services/tracing.ts \
+ *		src/debug/clean-pending-payments-for-dead-node.ts
+ */
+
+import { LedgerTransactionType, UnknownLedgerError } from "@domain/ledger"
+
+import { isUp } from "@services/lnd/health"
+import { params as unauthParams } from "@services/lnd/unauth"
+
+import { setupMongoConnection } from "@services/mongodb"
+import { LedgerService, translateToLedgerTx } from "@services/ledger"
+import { LockService } from "@services/lock"
+import { baseLogger } from "@services/logger"
+
+import { MainBook } from "@services/ledger/books"
+
+const PUBKEY = process.env.PUBKEY
```

Comment:
I think it would be more typically to pass this as an argument to the function, but it doesn't really matter.

Reply:
Thinking about this again, it's probably cleaner as a script arg because we don't know if we'd be overwriting some env variable with the env approach. Changed: https://github.com/GaloyMoney/galoy/pull/1583/commits/de7b1b16a86a2ee79c0ab57da00f1ca4f8e465b0